src/lib/components/organisms/ticker/index.jsx

```diff
@@ -19,11 +32,15 @@ export function Ticker({ maxItems = 20, onStateChange, children }) {
   const controlsRef = useRef()
 
   const [hideButtons, setHideButtons] = useState(false)
-
   const [expanded, setExpanded] = useState(false)
 
   const childArray = toChildArray(children)
 
+  const containerSize = useContainerSize(tickerRef)
+  const mobLandscapeW = 480
```

Comment:
Well this was actually for javascript reasons. I wanted to be able to have an `isMobile` in the code so I could split out the control components - different ones show at different breakpoints. Prior to this I had very nested jsx with lots of if statements and lots of CSS overrides. 

BUT - i do realise we have some "isMobile" work in the jsx and some in the CSS. This may be something we have to live with?

Reply:
Yep fair enough!